### DISCUSSION TITLE
post/2023/running-home-assistant-with-zigbee2mqtt-on-synology-nas-with-containers/

### DISCUSSION THREAD

**Discussion by giscus** [Jandev/janvnl-comments#3] [Announcements]
# post/2023/running-home-assistant-with-zigbee2mqtt-on-synology-nas-with-containers/

For those of you who are following me on Twitter, you might have seen my Philips Hue bridge was acting up in the past couple of months. Major bandwith usage, automations not being triggered at the appropriate times, and even the internet connectivity wasn’t working anymore. The bridge is also about 14 years old, so it might have been its time to shut down.
A relative cheap solution would be to buy a new Philips Hue bridge and set that up.

https://jan-v.nl/post/2023/running-home-assistant-with-zigbee2mqtt-on-synology-nas-with-containers/

**Comment by dmitriibeliakov** [comment]
Installing USB drivers is easier using community packages

Download USB drivers here. Ask ChatGPT for which architecture you have based on the CPU type - look it up in Control Panel -> Info Center. Find the right architecture and download it:
https://synocommunity.com/package/synokernel-usbserial

Then install using Package Center -> Manual install

**Reply by Jandev** [reply]
I didn't know the community packages also contained these drivers. Thanks for sharing!

Have you used these yourself? The downside of using the script & installing the drivers on boot is, well you have to install them each time the NAS reboots. While this isn't often, it's still annoying.
I _think_ these packages will be persisted across reboots, but would be great if you can confirm.

**Reply by dmitriibeliakov** [reply]
I only installed it yesterday. I'll test it. All this hassle with HA and Synology makes me think about buying a Raspberry PI or Home Assistant Green/yellow. Too much tinkering with Synology for my taste

**Reply by dmitriibeliakov** [reply]
All good, the community packages survive the restart. See the SMLIGHT Zigbee coordinator in USB 3-1

```
|__usb1          1d6b:0002:0404 09  2.00  480MBit/s 0mA 1IF  (Linux 4.4.180+ xhci-hcd xHCI Host Controller 0000:00:15.0) hub
  |__1-4         f400:f400:0100 00  2.00  480MBit/s 200mA 1IF  (Synology DiskStation 7F008A5B10CA6071)
|__usb2          1d6b:0003:0404 09  3.00 5000MBit/s 0mA 1IF  (Linux 4.4.180+ xhci-hcd xHCI Host Controller 0000:00:15.0) hub
|__usb3          1d6b:0002:0404 09  2.00  480MBit/s 0mA 1IF  (Linux 4.4.180+ etxhci_hcd-170202 Etron xHCI Host Controller 0000:02:00.0) hub
  |__3-1         10c4:ea60:0100 00  2.00   12MBit/s 100mA 1IF  (SMLIGHT SMLIGHT SLZB-06 f0b974a53c72ef11970b3b848fcc3fa0)
|__usb4          1d6b:0003:0404 09  3.00 5000MBit/s 0mA 1IF  (Linux 4.4.180+ etxhci_hcd-170202 Etron xHCI Host Controller 0000:02:00.0) hub
```

**Reply by Jandev** [reply]
Thanks for trying and getting back.
I'll be using these drivers from now on as it's much more convenient not to re-install them after reboots.

**Reply by dmitriibeliakov** [reply]
Sorry for the confusion, the drivers didn't work for me. The USB device is detected, but the drivers didn't work. I stopped using the USB connector and switched to the WiFi mode for the Zigbee coordinator. As you can see from above, I use SMLIGHT SLZB-06.
